src/Evolution/Systems/NewtonianEuler/Subcell/PrimitiveGhostData.hpp

```diff
@@ -41,4 +48,44 @@ class PrimitiveGhostDataOnSubcells {
   static Variables<prims_to_reconstruct_tags> apply(
       const Variables<prim_tags>& prims) noexcept;
 };
+
+/*!
+ * \brief Projects the mass density, velocity, and pressure to the subcells so
+ * they can be used for reconstruction.
```

Comment:
- As in the previous commit, can you clarify (or remove) the "Ghost" part of the name here? I'm still confused.
- Can you now also clarify the "ToSlice" part of the name? This just talks about projections and reconstructions, and I don't understand what will be sliced where.